doc/source/conf.py

```diff
@@ -172,10 +190,12 @@ def extract_example_links(
     list
         List of example links.
     """
-    g = Github()
+    access_token = os.getenv("GITHUB_ACCESS_TOKEN")
+    g = Github(access_token)
```

Comment:
Do we really need a token for this?

Reply:
no, i needed it locally since i build a couple of times, i am removing it.